src/test/java/com/inventorysystem/inventory/InventoryApplicationTests.java

```diff
@@ -7,68 +7,64 @@
 import com.inventorysystem.inventory.service.ProductServiceStub;
 import com.inventorysystem.inventory.dto.Product;
 import com.inventorysystem.inventory.dao.IProductDAO;
-import com.inventorysystem.inventory.dao.ProductDAOStub;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import com.inventorysystem.inventory.dao.ProductDAOStub;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
 @SpringBootTest
 class InventoryApplicationTests {
     private IProductService productService;
-    private Product product = new Product();
-    private Product product2 = new Product();
+    private Product productPaperCup = new Product();
+    private Product productDairyButter = new Product();
```

Comment:
Very good.  Qualify the name of the product with something meaningful, so that the variable name is descriptive.